xbmc/Util.h

```diff
@@ -227,6 +227,10 @@ class CUtil
                                    const std::string& file,
                                    const std::string& destname = "");
 
+  /** \brief Delete a file with confirmation unless setting is off.
+  */
+  static bool FileDeleteWithConfirm(const CFileItemPtr& item);
```

Comment:
One more time: 

1) This function should be moved to `CFileUtils`.
2) Replace `CFileItemPtr` with `std::shared_ptr>CFileItem>`.

Reply:
Done